sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/StoreType.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// Time Series Insights store type.
+    /// </summary>
+    public class StoreType
+    {
+        private readonly string _value;
+        private const string WarmStoreValue = "WarmStore";
+        private const string ColdStoreValue = "ColdStore";
+
+        internal StoreType(string value)
+        {
+            _value = value;
+        }
+
+        /// <summary>
+        /// Time Series Insights warm store.
+        /// </summary>
+        public static StoreType WarmStore { get; }  = new StoreType(WarmStoreValue);
```

Comment:
There's an extra space before the = sign.

Reply:
I wonder if there is a way to do ctrl+k ctrl+d on the entire solution instead of a single file. Will look that up.